themes-default/slim/views/vue-components/show-selector.mako

```diff
@@ -28,14 +28,14 @@ Vue.component('show-selector', {
     computed: Object.assign(store.mapState(['config', 'shows']), {
         showLists() {
             const { config, shows } = this;
-            const { sortArticle } = config;
+            const { animeSplitHome, sortArticle } = config;
             const lists = [
                 { type: 'Shows', shows: [] },
                 { type: 'Anime', shows: [] }
             ];
 
             shows.forEach(show => {
-                const type = show.config.anime === true ? 1 : 0;
+                const type = animeSplitHome && show.config.anime ? 1 : 0;
```

Comment:
Change this to `Number(animeSplitHome && show.config.anime)` as it'll return 1/0 depending on the boolean passed.